app/src/main/java/com/brainwallet/ui/screens/ready/ReadyScreen.kt

```diff
@@ -33,23 +33,27 @@ import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.text.withStyle
 import androidx.compose.ui.unit.dp
-import androidx.core.widget.TextViewCompat.AutoSizeTextType
 import com.brainwallet.R
-import com.brainwallet.data.model.AppSetting
 import com.brainwallet.navigation.OnNavigate
 import com.brainwallet.navigation.Route
 import com.brainwallet.navigation.UiEffect
 import com.brainwallet.ui.composable.BorderedLargeButton
 import com.brainwallet.ui.composable.BrainwalletScaffold
 import com.brainwallet.ui.composable.BrainwalletTopAppBar
+import org.koin.compose.koinInject
 
 
 @Composable
 fun ReadyScreen(
-    onNavigate: OnNavigate
+    onNavigate: OnNavigate,
+    viewModel: ReadyViewModel = koinInject()
 ) {
     val context = LocalContext.current
 
+    LaunchedEffect(Unit) {
```

Comment:
@andhikayuana this is looking really good. Lets make it even sooner. So, when the user **launches the app,** sync start....So not when they press the "Ready" button but as soon as they **see** the welcome Screen
We also need to fix the workaround.  Currently, a user onboarding who closes this screen can bypass the Set Seed words and never write them down.
https://github.com/gruntsoftware/internal/issues/101

Reply:
noted